.github/workflows/project-codeflare-release.yml

```diff
@@ -38,6 +38,9 @@ on:
         description: 'Owner of target community-operators-prod repository used to open a PR against'
         required: true
         default: 'redhat-openshift-ecosystem'
+      rhoai-release-version:
+        description: "RHOAI Release version for updating Component Release Matrix Versions Info "
+        required: true
```

Comment:
`quay-organization`, `is-stable` and `codeflare-repository-organization` seem like they can all be hard coded. 
I wouldn't ever see the need to release an unstable version of the stack and the two organisations can be hard coded and worst case scenario a PR can be merged in future to change them